docs/scripts/autogenerate-docs.py

```diff
@@ -85,7 +83,7 @@ def generate_cli_docs():
         shutil.rmtree(generated_cli_docs_dir)
 
     # Generate the CLI docs using "juju documentation" command.
-    subprocess.run(['juju', 'documentation', '--split', '--no-index', '--out', generated_cli_docs_dir],
+    subprocess.run(['go', 'run', 'scripts/docgen.go', '--split', '--no-index', '--out', generated_cli_docs_dir],
                    check=True)
 
```

Comment:
I'm not sure what's happening here. I know in 4 we were using a script but the script seems to have been moved under `docs/scripts` (see next file in this PR), so the path given here wouldn't work. I'm probably missing something but wanted to draw your attention to this either way.

Reply:
I got confused. All is well. Thanks!

Reply:
The scripts bit in the path is needed because it is invoked from one level up. `make docs-run` does the right thing